pkg/interfaces/contracts/standalone-utils/IBalancerFeeBurner.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity ^0.8.24;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { IProtocolFeeBurner } from "./IProtocolFeeBurner.sol";
+
+interface IBalancerFeeBurner is IProtocolFeeBurner {
+    /**
+     * @notice Steps for the burn path
+     * @param pool The pool for the swap
+     * @param tokenOut The token out of the swap
+     */
+    struct SwapPathStep {
+        address pool;
+        IERC20 tokenOut;
+    }
+
+    /**
+     * @notice Data for the burn hook
+     * @param pool The pool the fees came from (only used for documentation in the event)
+     * @param sender The sender of the call. In most cases, this is the sweeper.
+     * @param feeToken The token collected from the pool
+     * @param feeTokenAmount The number of fee tokens collected
+     * @param targetToken The desired target token (token out of the swap)
+     * @param minAmountOut The minimum amount out for the swap
+     * @param recipient The recipient of the swap proceeds
+     * @param deadline Deadline for the burn operation (i.e., swap), after which it will revert
+     */
+    struct BurnHookParams {
+        address pool;
+        address sender;
+        IERC20 feeToken;
+        uint256 feeTokenAmount;
+        IERC20 targetToken;
+        uint256 minAmountOut;
+        address recipient;
+        uint256 deadline;
+    }
+
+    /// @notice Burn path not set for the fee token
+    error BurnPathNotExists();
+
+    /// @notice The last token in the path is not the same as the target token
+    error TargetTokenInPathNotTheSame();
```

Comment:
Consider naming this more explicitly. Like `FinalTokenOutNotTarget` or `TokenOutTargetMismatch`